<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>508</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main</string>
  </property>
  <widget class="QWidget" name="WCentral">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="Nav">
      <item row="0" column="1">
       <widget class="QPushButton" name="NavDelete">
        <property name="text">
         <string>Eliminar</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditDelete"/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="NavEdit">
        <property name="styleSheet">
         <string notr="true">QPushButton:disabled {
                background-color: white;  /* Color de fondo cuando est치 deshabilitado */
                color: black;
}
</string>
        </property>
        <property name="text">
         <string>Editar</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MailMessageNew"/>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="NavSave">
        <property name="styleSheet">
         <string notr="true">QPushButton::disabled {
                background-color: white;  /* Color de fondo cuando est치 deshabilitado */
                color: black;
            }
</string>
        </property>
        <property name="text">
         <string>Guardar</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="WForm">
      <item row="0" column="1">
       <layout class="QGridLayout" name="Form">
        <item row="7" column="1">
         <widget class="QPushButton" name="FAdd">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1200</width>
            <height>500</height>
           </size>
          </property>
          <property name="text">
           <string>Agregar</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::ListAdd"/>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="FNLable">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="FName">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPlainTextEdit" name="FDText">
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="FDesc">
          <property name="text">
           <string>Descripci칩n:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="FPriority">
          <property name="text">
           <string>Prioridad:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="FOptions">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox::item {
                height: 300px;
}</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Baja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Moderada</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Urgente</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="Table">
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Tareas</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prioridad</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descripci칩n</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
